R/commonAnovaBayesian.R

```diff
@@ -171,6 +171,15 @@
             return(msg)
           }
           return(NULL)
+        },
+        redundantParticipantColumn = function() {
+          
+          if (nlevels(dataset[.BANOVAsubjectName]) == nlevels(dataset[target])) {
+            return(gettextf("Duplicate participant column added to model terms, %s", target))
+          } else {
```

Comment:
So I didn't test this in JASP, but:
```r
df <- data.frame(a = gl(5, 2, labels = letters[1:5]))
df
#   a
#1  a
#2  a
#3  b
#4  b
#5  c
#6  c
#7  d
#8  d
#9  e
#10 e

nlevels(df["a"])
# [1] 0
nlevels(df[["a"]])
# [1] 5
```
did you mean to do `dataset[[.BANOVAsubjectName]]` and `dataset[[target]]`? So `[[` rather than `[`?

Reply:
ah yes, good catch! I did test it in jasptools, but i guess i was just testing 0 == 0...